processor/src/services/adyen-payment.service.ts

```diff
@@ -73,9 +83,7 @@ export class AdyenPaymentService extends AbstractPaymentService {
   private refundPaymentConverter: RefundPaymentConverter;
 
   constructor(opts: AdyenPaymentServiceOptions) {
-    super(opts.ctCartService, opts.ctPaymentService);
-    this.ctCartService = opts.ctCartService;
-    this.ctPaymentService = opts.ctPaymentService;
```

Comment:
There is no need to set these values manually. The `AbstractPaymentService` already does this.